sql/core/src/main/scala/org/apache/spark/sql/package.scala

```diff
@@ -46,6 +46,6 @@ package object sql {
    * Type alias for [[DataFrame]]. Kept here for backward source compatibility for Scala.
    * @deprecated As of 1.3.0, replaced by `DataFrame`.
    */
-  @deprecated("1.3.0", "use DataFrame")
+  @deprecated("use DataFrame instead", "1.3.0")
```

Comment:
Could you put the warning message just like "Use DataFrame" as with any other warning messages.